lxc/remote.go

```diff
@@ -530,22 +530,25 @@ func (c *cmdRemoteAdd) run(cmd *cobra.Command, args []string) error {
 					return err
 				}
 
-				// Continue with adding the remote if digest matches, or the user
-				// confirmed a fingerprint.
-				if string(line) == digest || strings.ToLower(string(line[0])) == i18n.G("y") {
-					break
-				}
+				// Check length before accessing line [0]
```

Comment:
This comment tells readers what the check is doing, but it doesn't address _why_ this check is required. 

```suggestion
				// Check length before accessing line to prevent runtime panic.
```